### DISCUSSION TITLE
I cannot open zepp simulator

### DISCUSSION THREAD

**Discussion by masm611** [zepp-health/zeppos-forum#192] [Q&A]
[firmware_simulator]: simulator opening...
[firmware_simulator]: .\qemu_win\qemu-system-arm.exe: -drive file=C:\Users\申陈旧\.zepp\emulator_cache\0a6df092a20271de6fb01bf1082d107d\norflash.bin,bus=0,unit=0,format=raw: Could not open 'C:\Users\申陈旧\.zepp\emulator_cache\0a6df092a20271de6fb01bf1082d107d\norflash.bin': 系统找不到指定的路径。

how can i fix this bug?

**Comment by htoooth** [comment]
@arduino-cpu  maybe you will try to change  path from Chinese  to English.

**Comment by TediPower** [comment]
[firmware_simulator]: simulator opening...
[firmware_simulator]: .\qemu_win\qemu-system-arm.exe: -net tap,ifname=tun: Device 'tap' could not be initialized

how can i fix this bug?

**Reply by MyoungXUE** [reply]
Hello, it seems that the virtual network card is not installed and configured correctly, please refer to https://docs.zepp.com/docs/guides/tools/simulator/setup/.
